integration-tests/micrometer-jmx/src/main/resources/application.properties

```diff
@@ -0,0 +1,11 @@
+
+quarkus.log.category."io.quarkus.micrometer".level=DEBUG
+quarkus.log.category."io.quarkus.bootstrap".level=INFO
+quarkus.log.category."io.quarkus.netty".level=INFO
+quarkus.log.category."io.quarkus.resteasy.runtime".level=INFO
+
+quarkus.log.category."io.netty".level=INFO
+quarkus.log.category."org.apache".level=INFO
+
+quarkus.micrometer.export.prometheus.enabled=false
```

Comment:
We need to explicitly disable the prometheus export when the dependency isn't present?

Reply:
We shouldn't. I may have just to be sure.. this might also be left over from how things were structured in my other project. Good observation.

Reply:
fixed.